packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -5,9 +5,10 @@ import { uiColors } from '@leafygreen-ui/palette';
 import { ulStyleOverrides, SIDE_OFFSET } from './styles';
 
 const sideNavLabelStyle = css`
-  color: ${uiColors.green.base};
+  font-size: 11px;
```

Comment:
So this is more push-back against the design than this PR, but looking at this in context right now on Cloud, `11px` is borderline legible. I really think `12px` is the smallest we should go for something that will show up on nearly every page of every application.

Example: I have a moderate prescription for glasses (nearsighted + astigmatism), and without my glasses the letters of the side nav group header at 11px start to blur into each other, about one foot from the monitor.

Take it or leave it, just a thought.

Reply:
Rob is working on typography guidelines which will give us the justification to change this. As of now, we should probably stick with the mocks on size, though I did work with Nelle to increase the contrast.

We would need to make some larger modifications to the spacing around the label to maintain the visual hierarchy to make the change to a larger font-size, but I do agree with you. I think we should go forward with this.

That said, if we add `letter-spacing: 0.3px;` to the component, does this address your concerns at all?

Reply:
Sure thing. If this is being addressed overall with guidelines, I'm good with whatever we need for now.